addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4835,7 +4835,7 @@ msgstr ""
 
 #: xbmc/dialogs/GUIDialogGamepad.cpp
 msgctxt "#12331"
-msgid "press OK, or Back to cancel"
+msgid "press OK, or back to cancel"
```

Comment:
not sure I like it, because "back" is referring to a button on the gamepad as it seems, just like OK. So probably put them into quotes and keep it uppercased? One would have to check the actual usage though, because if it's some sort of dialog message, the entire paragraph would have to start uppercase as well